engine/runtime/src/main/java/org/enso/ClassLoaderConstants.java

```diff
@@ -13,10 +13,7 @@ public class ClassLoaderConstants {
    * consistent.
    */
   public static final List<String> CLASS_DELEGATION_PATTERNS =
-      List.of("org.graalvm", "java", "org.slf4j", "ch.qos", "io.helidon");
-
-  public static final List<String> RESOURCE_DELEGATION_PATTERNS =
-      List.of("org.slf4j", "ch.qos", "io.helidon");
+      List.of("org.graalvm", "java", "org.slf4j", "ch.qos", "io.helidon", "org.enso.profiling", "org.enso.syntax2", "org.enso.ydoc");
```

Comment:
Only one field is necessary here. From Classloader's perspective, a class is a special case of a resource when looking up via a binary name.